frameworks/elastic/universe/config.json

```diff
@@ -34,6 +34,11 @@
           "description": "Elasticsearch deploy strategy. [parallel, serial]",
           "type": "string",
           "default": "parallel"
+        },
+        "mesos_api_version": {
```

Comment:
It's apparently needed to make strict work: https://github.com/mesosphere/dcos-commons/pull/1013/files#diff-a4edd5b45d2ef4fda8fd5a8e90348736R115